numpy/polynomial/_polybase.py

```diff
@@ -123,6 +123,10 @@ def _line(self):
         pass
 
     @abstractmethod
+    def _interp(self):
+        pass
```

Comment:
Given you don't seem willing to implement this in the other polynomial classes, you should just remove the `@abstractmethod`, and implement it to `raise NotImplemented`